core/silkworm/state/intra_block_state.cpp

```diff
@@ -307,7 +307,8 @@ void IntraBlockState::revert_to_snapshot(const IntraBlockState::Snapshot& snapsh
 }
 
 void IntraBlockState::finalize_transaction() {
-    for (auto& [_, storage] : storage_) {
+    for (auto& x : storage_) {
```

Comment:
You could have left the structured binding and simply add `(void)address` into the loop.
Nevertheless the cost is the same as in actual implementation `x` holds two references (first and second).

Just for reference. When GCC8 this also prevents the triggering of warning error
```
for ([[maybe_unused]] auto& [address, storage] : storage_) { 
 // Whatever
}
```